practice_questions/prime_numbers_range.py

```diff
@@ -1,14 +1,30 @@
-a = int(input("enter the start  number: "))
-b = int(input("enter the  end number: "))
+# a = int(input("enter the start  number: "))
+# b = int(input("enter the  end number: "))
 
 
-if a <= 1:
-    print(a, "is not a prime number")
-elif a > 1:
+# if a <= 1:
+#     print(a, "is not a prime number")
+# elif a > 1:
 
-    for i in range(a,b//2):
-        if a % i == 0:
-            # print()
-            break
-    else:
-        print(a, 'is a prime number')
\ No newline at end of file
+#     for i in range(a,b//2):
+#         if a % i == 0:
+#             # print()
+#             break
+#     else:
+#         print(a, 'is a prime number')
+
+# Python program to display all the prime numbers within an interval
+
+lower = 0
+upper = 10
+
+# print("Prime numbers between", lower, "and", upper, "are:")
+
+for num in range(lower, upper + 1):
```

Comment:
Add this in a function and call it; This items of work is just for you to work on the comment and change the code and again to git add that file, commit and push. And i will make the review the change and merge it